src/components/K8sContainersOverview/K8sContainersOverview.js

```diff
@@ -150,6 +149,14 @@ const K8sContainersOverview = (props) => {
       setMemorySaturationTimeLabels(updatedMemorySaturationValues.timeLabels);
       setMemorySaturationDatasets(updatedMemorySaturationValues.dataset);
     }
+  };
+
+  // update metric values with the results of calling the above functions and update state
+  // BUG WITH SET INTERVAL: when you switch to a new pod, the previously selected pods' linecharts are still being rendered every 3 seconds**/
```

Comment:
Does this bug still exist?

Reply:
yes, this bug still exists, so the setInterval function and cleanup are commented out to prevent the bug from running